src/main/java/com/epam/grid/engine/provider/utils/slurm/healthcheck/ShowConfigCommandParser.java

```diff
@@ -0,0 +1,159 @@
+    private static final String CANT_FIND_CONNECTION_MESSAGE = "Can`t find connection via specified port";
+    private static final String DATE_FORMAT_CHECK_TIME = "MM/dd/yyyy HH:mm:ss:";
+    private static final String DATE_FORMAT_START_TIME = "yyyy-MM-dd'T'HH:mm:ss";
+    private static final String BOOT_TIME = "BOOT_TIME";
+    private static final Long NOT_PROVIDED = 99999L;
+    private static final String DOUBLED_SPACES_REGEX = "\\s+";
+
+    public static HealthCheckInfo parseShowConfigResult(final CommandResult commandResult) {
+        validateShowConfigResponse(commandResult);
+        final List<String> stdOut = commandResult.getStdOut();
+        final StatusInfo statusInfo = parseStatusInfo(stdOut);
+        return HealthCheckInfo.builder()
+                .statusInfo(statusInfo)
+                .startTime(getStartTime(stdOut))
+                .checkTime(getCheckTime())
+                .build();
+    }
+
+    private static void validateShowConfigResponse(final CommandResult commandResult) {
+        if (CollectionUtils.isEmpty(commandResult.getStdOut())) {
+            throw new GridEngineException(HttpStatus.NOT_FOUND,
+                    String.format("Slurm error during health check. %nexitCode = %d %nstdOut: %s %nstdErr: %s",
+                            commandResult.getExitCode(), commandResult.getStdOut(), commandResult.getStdErr())
+            );
+        }
+        if (checkCantFindConnectionCase(commandResult)) {
+            throw new GridEngineException(HttpStatus.NOT_FOUND, CANT_FIND_CONNECTION_MESSAGE);
+        }
+    }
+
+    private static boolean checkCantFindConnectionCase(final CommandResult commandResult) {
+        return commandResult.getExitCode() != 0
+                && commandResult.getStdOut().get(0).contains(CANT_FIND_CONNECTION)
+                && !commandResult.getStdErr().isEmpty();
+    }
+
+    private static StatusInfo parseStatusInfo(final List<String> stdOut) {
+        final long statusCode = parseStatusCode(stdOut);
+        return StatusInfo.builder()
+                .code(statusCode)
+                .status(parseStatus(statusCode))
+                .info(parseInfo(stdOut))
+                .build();
+    }
+
+    private static long parseStatusCode(final List<String> stdOut) {
+        final long statusCode;
+        final List<String> statusString = List.of(stdOut.get(stdOut.size() - 1)
+                .replaceAll(DOUBLED_SPACES_REGEX, SPACE)
+                .split(SPACE));
+        final String status = statusString.get(statusString.size() - 1);
+        switch (status) {
+            case UP_STATE:
+                statusCode = 0L;
+                break;
+            case DOWN_STATE:
+                statusCode = 2L;
+                break;
+            default:
+                statusCode = NOT_PROVIDED;
+                break;
+        }
+        return statusCode;
+    }
+
+    private static GridEngineStatus parseStatus(final long status) {
```

Comment:
here we don't parse anything but just get by id

Reply:
method's name was changed